app/services/participant_outcomes/create.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+module ParticipantOutcomes
+  class Create
+    include ActiveModel::Model
+    include ActiveModel::Attributes
+
+    STATES = %w[passed failed].freeze
+    COURSE_IDENTIFIERS = %w[npq-early-headship-coaching-offer npq-additional-support-offer].freeze
+    COMPLETION_DATE_FORMAT = /\d{4}-\d{2}-\d{2}/
+
+    attr_reader :created_outcome
+
+    attribute :lead_provider
+    attribute :participant
+    attribute :course_identifier
+    attribute :state
+    attribute :completion_date
+
+    validates :lead_provider, presence: true
+    validates :participant, presence: true
+    validates :course_identifier, inclusion: { in: COURSE_IDENTIFIERS }
+    validates :state, inclusion: { in: STATES }
+    validates :completion_date, presence: true, format: { with: COMPLETION_DATE_FORMAT }
```

Comment:
I'm seeing an exception being raised when I'm calling the service with an invalid date ie:
```
ParticipantOutcomes::Create.new(
  lead_provider: Application.accepted.last.lead_provider,
  course_identifier: Application.accepted.last.course.identifier,
  participant: Application.accepted.last.user,
  completion_date: "2022-13-01",
  state: "passed",
).valid?
```

Reply:
Good spot, I'll update 👍